tests/test_syntax/blocks/test_headers.py

```diff
@@ -583,7 +583,7 @@ def test_hash_gt6_open_trailing_space(self):
         )
 
     # TODO: Possibly change the following behavior. While this follows the behavior
-    # of markdown.pl, it is rather uncommon and not nessecarily intuitive.
+    # of markdown.pl, it is rather uncommon and not neccesarily intuitive.
```

Comment:
I think it's still wrong.

```suggestion
    # of markdown.pl, it is rather uncommon and not necessarily intuitive.
```

Reply:
Thanks, fixed.